base/applications/regedit/framewnd.c

```diff
@@ -1146,12 +1146,7 @@ FreeObjectPicker(IN IDsObjectPicker *pDsObjectPicker)
     pDsObjectPicker->lpVtbl->Release(pDsObjectPicker);
 }
 
-/*******************************************************************************
- *
- *  FUNCTION: _CmdWndProc(HWND, unsigned, WORD, LONG)
- *
- *  PURPOSE:  Processes WM_COMMAND messages for the main frame window.
- *
+/* PURPOSE: Processes WM_COMMAND messages for the main frame window.
```

Comment:
```suggestion
/*
 * PURPOSE: Processes WM_COMMAND messages for the main frame window.
```

Reply:
feels like a wasted line to me, but if it will make an approval appear, then fine. Committed.